lib/charms/opensearch/v0/opensearch_plugins.py

```diff
@@ -458,6 +528,12 @@ def config(self) -> OpenSearchPluginConfig:
             secret_entries_to_del = [...]|{...},
         )
         """
+        if not self._extra_config.get("access-key") and not self._extra_config.get("secret-key"):
+            raise OpenSearchPluginMissingConfigError(
+                "Missing AWS access-key and secret-key configuration",
+                affected_configs=["access-key", "secret-key"],
+                only_log=True,
```

Comment:
This is not needed, the message `Missing AWS access-key and secret-key configuration` should already be enough.